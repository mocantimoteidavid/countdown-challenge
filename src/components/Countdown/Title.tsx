import styled from "styled-components"

const Title = styled.h3`
  text-transform: uppercase;
  letter-spacing: 3px;
  font-size: 14px;
  margin-bottom: 6px;
  font-weight: 400;
`

export default Title
