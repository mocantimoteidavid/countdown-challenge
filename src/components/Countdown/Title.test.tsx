import { render } from "@testing-library/react"
import Title from "./Title"

describe("Title", () => {
  it("renders with correct styling", () => {
    const { container } = render(<Title>Alladin</Title>)

    expect(container.firstChild).toMatchSnapshot()
  })
})
