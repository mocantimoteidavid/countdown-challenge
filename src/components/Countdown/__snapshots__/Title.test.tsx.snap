// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title renders with correct styling 1`] = `
.c0 {
  text-transform: uppercase;
  -webkit-letter-spacing: 3px;
  -moz-letter-spacing: 3px;
  -ms-letter-spacing: 3px;
  letter-spacing: 3px;
  font-size: 14px;
  margin-bottom: 6px;
  font-weight: 400;
}

<h3
  class="c0"
>
  Alladin
</h3>
`;
